### ISSUE TITLE
Optional shadow dom.  

### ISSUE THREAD

**Issue by deebloo** [deebloo/lit-kit#26]
Shadow dom gives us native encapsulation and content projection but at the moment has some difficulties with screen readers. Because of this shadow dom should be optional. This will mean fixing an issue where the template is applied in the constructor rather than connectedCallback
